system/libuv/Makefile

```diff
@@ -69,12 +69,12 @@ CSRCS += posix-hrtime.c
 CSRCS += posix-poll.c
 CSRCS += uv-data-getter-setters.c
 CSRCS += version.c
-ifneq ($(CONFIG_LIBUV_UTILS_TEST),)
+ifeq ($(CONFIG_LIBUV_UTILS_TEST),)
```

Comment:
The logic for `idna.c` is changed. Is it expected?

Reply:
Yes, both idna.c and strscpy.c will be included directly when test is enabled. Please reference here:
https://github.com/libuv/libuv/blob/master/test/test-idna.c#L23
https://github.com/libuv/libuv/blob/master/test/test-strscpy.c#L27

Reply:
Sure. Just want to confirm that this is "as expected".